openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1097,3 +1098,38 @@ def verify_if_more_than_n_percentage_pod_restarted(
             msg = "Out of {} pods {} pods restarted {}".format(
                 len(pods_new), len(pods_restart), pods_restart)
             raise AssertionError(msg)
+
+    def validate_glusterfsd_memory_usage(self, size_limit):
```

Comment:
Any specific reason for defining this function, As this is already a part of the base class?

Reply:
It was because of the wrong rebase.